.github/workflows/basic.yml

```diff
@@ -42,17 +43,34 @@ jobs:
             compiler: {c: gcc, cxx: g++}
             shared_library: 'OFF'
             os_provider: 'OFF'
+          # TODO: Enable sanitizer checks with pool tracking enabled. There's
+          # a leak reported by sanitizer in jemalloc pool implementation that
+          # points to a tracker_value_t not being free'd.
```

Comment:
Please add the link to this issue here:
https://github.com/oneapi-src/unified-memory-framework/issues/147